playbook-website/app/views/pages/kit_show.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="pb--kit-show <%=@kit%>-kit">
-  <%= pb_rails("title", props: { text: pb_kit_title(@kit), tag: "h1", size: 1 }) %>
+  <%= pb_rails("title", props: { text: pb_kit_title(@kit), tag: "h1", size: 1, margin: "xl" }) %>
```

Comment:
Why was this added?

Reply:
Thats was @jasoncypret suggestion so we could use `top: 0`.